client/src/components/modal-forms/NewPSIForm.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+import { Button } from '../generic';
+import { Box } from '@material-ui/core';
+import { RenderTextField, RenderSelectField } from '../fields';
+import { Field, Formik, Form as FormikForm } from 'formik';
+import { CreatePSISchema, healthAuthorities } from '../../constants';
+
+export const NewPSIForm = ({ initialValues, onSubmit, onClose }) => {
+  return (
+    <Formik initialValues={initialValues} validationSchema={CreatePSISchema} onSubmit={onSubmit}>
+      {({ submitForm, values }) => (
```

Comment:
See SonarCloud scan (code smell: remove unused values param or rename to _values)- I think that could be cleaned up!

Reply:
I think `_values` is a mistake for interpreting this bit as function parameters, in this case where it's a destructured object, removal is the best option if it's unused